### PR TITLE
Rename package to wagtail_wordpress_import

### PR DESCRIPTION
**Pull request by nickmoreton**

### CODE REVIEW

File: wagtail_wordpress_import/__init__.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+default_app_config = "wagtail_wordpress_import.apps.WagtailXmlimportAppConfig"
```
Comment (kaedroho) [inline wagtail_wordpress_import/__init__.py]: Needs updating

File: wagtail_wordpress_import/apps.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
 from django.apps import AppConfig
 
 class WagtailXmlimportAppConfig(AppConfig):
```
Comment (kaedroho) [inline wagtail_wordpress_import/apps.py]: Class name needs updating

File: wagtail_wordpress_import/test/__init__.py [hunk 3]
```diff
@@ -0,0 +1 @@
+default_app_config = "wagtail_wordpress_import.test.apps.WagtailXmlimportTestAppConfig"
```
Comment (kaedroho) [inline wagtail_wordpress_import/test/__init__.py]: Needs updating

File: wagtail_wordpress_import/test/apps.py [hunk 4]
```diff
@@ -2,6 +2,6 @@
 
 
 class WagtailXmlimportTestAppConfig(AppConfig):
```
Comment (kaedroho) [inline wagtail_wordpress_import/test/apps.py]: Class name needs updating

**Review by kaedroho — COMMENTED** [review]
Just need to update the name of the AppConfig classes but otherwise looks good to me!

**Comment by nickmoreton** [comment]
Hi Karl These changes are all in place now.

https://github.com/torchbox/wagtail-wordpress-import/pull/28/commits/fc7b8e0ea86b336089ecbfe3873cd4eb2bbfbec4

**Pull request merged**